public/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "short_name": "Scrumlr",
```

```suggestion
  "short_name": "scrumlr.io",
```
I prefer the lowercase title with our TLD included. Just personal preference :)